feat: Implement the prompt module for time (#138)
starship/starship: docs/config/README.md

symbol = "⚙️ "
```

## Time

The `time` module shows the current **local** time.

::: tip
For the `format` configuration value, take a look [here](https://docs.rs/chrono/0.4.7/chrono/format/strftime/index.html) to see what you can pass in.
:::

### Options

| Variable   | Default | Description                                  |
| ---------- | ------- | -------------------------------------------- |
| `12hr`     | `false` | Enables 12 hour formatting                   |
| `format`   | `''`    | Enables the user to set a custom time format |

> | `disabled` | `false` | Disables the `username` module.              |

Reviewer: Should this say "Disables the `time` module."?

```suggestion
| `disabled` | `false` | Disables the `time` module.              |
```

Author: It totally should @LinusU ... thanks for reviewing!

symbol = "⚙️ "
```

## Time

The `time` module shows the current **local** time.

> ::: tip

Reviewer 2: I would personally suggest putting the details about formatting in the description above. I think the tip box would be better suited to remind the user that this module is "disabled by default".

Author: Done! (https://github.com/starship/starship/pull/138/commits/7727b90ddad41784aa6ea39421bc28a578bd3bdc)

For the `format` configuration value, take a look [here](https://docs.rs/chrono/0.4.7/chrono/format/strftime/index.html) to see what you can pass in.
:::

### Options

| Variable   | Default | Description                                  |
| ---------- | ------- | -------------------------------------------- |
| `12hr`     | `false` | Enables 12 hour formatting                   |

> | `format`   | `''`    | Enables the user to set a custom time format |

Reviewer 2: ```suggestion
| `format`   | `''`    | Accepts a [chrono format string](https://docs.rs/chrono/0.4.7/chrono/format/strftime/index.html) to customize the time formatting |
```

symbol = "⚙️ "
```

## Time

The `time` module shows the current **local** time.

> The `format` configuration value is used by the `chrono` crate to control how the time is displayed. Take a look [at the chrono strftime docs](https://docs.rs/chrono/0.4.7/chrono/format/strftime/index.html) to see what options are available.

Reviewer 2: I'd suggest adding a link here:

```suggestion
The `format` configuration value is used by the [`chrono`](https://crates.io/crates/chrono) crate to control how the time is displayed. Take a look [at the chrono strftime docs](https://docs.rs/chrono/0.4.7/chrono/format/strftime/index.html) to see what options are available.
```
